python/lsst/analysis/ap/nb_utils.py

```diff
@@ -131,22 +121,10 @@ def compare_sources(butler1, butler2, query1, query2,
     cutout_path2 : `str`, optional
         Base path to store cutouts for sources unique to dataset2.
         Must be supplied if make_cutouts is True.
-    chunk_size1 : `int`, optional
-        Chunk size to use for output directories of cutouts unique to
-        dataset 1. If using an existing set of cutouts made with
-        plotImageSubtractionCutouts, be sure to set this value to
-        match what was used when making the set. Then the code will
-        detect the existing files and not attempt to remake them.
-        If not using a preexisting set, this value can be set to
-        any power of 10, or omitted entirely.
-    chunk_size2 : `int`, optional
-        Chunk size to use for output directories of cutouts unique to
-        dataset 2. If using an existing set of cutouts made with
-        plotImageSubtractionCutouts, be sure to set this value to
-        match what was used when making the set. Then the code will
-        detect the existing files and not attempt to remake them.
-        If not using a preexisting set, this value can be set to
-        any power of 10, or omitted entirely.
+    cutout_config1 : `dict` [`str`], optional
+        Config overrides to apply to cutout plotter for dataset 1.
```

Comment:
Please reference `plotImageSubtractionCutouts.PlotImageSubtractionCutoutsConfig()` as the source of the config.